docs/toc.html

```diff
@@ -52,7 +52,8 @@
                 <li><a href="#streamsconfigs">3.6 Kafka Streams Configs</a>
                 <li><a href="#adminclientconfigs">3.7 AdminClient Configs</a>
                 <li><a href="#mirrormakerconfigs">3.8 MirrorMaker Configs</a>
-                <li><a href="#systemproperties">3.9 System Properties</a>
+                <li><a href="#remotestorageconfigs">3.9 Tier Storage Configs</a></li>
```

Comment:
"Remote Storage Configs"

Also please move this to 3.10 (instead of 3.9) so that existing hyperlinks for "System Properties" continue working properly.